### PR TITLE
[4] Remove unused imports

### PR DESCRIPTION
**Pull request by PhilETaylor**
Code review

### CODE REVIEW

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on fdd14471cd6d8c0b3910c28a2a2a5fb014719e32<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33948">issues.joomla.org/tracker/joomla-cms/33948</a>.</sub>

**Comment by joomdonation** [comment]
@PhilETaylor Could you synchronize this branch with latest 4.0-dev? Some changes in this PR was already included in your other merged PRs.

**Comment by PhilETaylor** [comment]
Done - went from 10 files to 6

**Comment by joomdonation** [comment]
Reviewed once more time again after synchronizing with 4.0-dev. RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33948">issues.joomla.org/tracker/joomla-cms/33948</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9eecfe85bcf3034eab36418250fb6895a9c32b54<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33948">issues.joomla.org/tracker/joomla-cms/33948</a>.</sub>

**Comment by richard67** [comment]
Thanks!

File: libraries/src/Service/Provider/Session.php [hunk 1]
```diff
@@ -34,7 +34,6 @@
 use Joomla\Session\HandlerInterface;
 use Joomla\Session\SessionEvents;
 use Joomla\Session\SessionInterface;
-use Joomla\Session\Storage\RuntimeStorage;
```
Comment (richard67) [inline libraries/src/Service/Provider/Session.php:37]: It seems this one here broke API test for invalid credentials, see https://ci.joomla.org/artifacts/joomla/joomla-cms/4.0-dev/33724/system-tests/44297/BasicCest.testWrongCredentials.fail.html . Strange is that for the PR they hadn't failed.

Comment (PhilETaylor) [reply libraries/src/Service/Provider/Session.php:37]: related to https://github.com/joomla/joomla-cms/pull/34043 maybe ?

Comment (PhilETaylor) [reply libraries/src/Service/Provider/Session.php:37]: fixed with https://github.com/joomla/joomla-cms/pull/34149

Comment (richard67) [reply libraries/src/Service/Provider/Session.php:37]: yes that was it.

Comment (PhilETaylor) [reply libraries/src/Service/Provider/Session.php:37]: Im guessing both PR's were right at the time .. but due to the quick turn around of PRs and merging they overlapped and this caused the issue. Sorry.

Comment (richard67) [reply libraries/src/Service/Provider/Session.php:37]: Can happen. At least we've found it.

**Pull request merged**